src/main/java/uk/gov/cabinetoffice/csl/service/messaging/model/registeredLearners/RegisteredLearnersOrganisationDelete.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.cabinetoffice.csl.service.messaging.model.registeredLearners;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@Getter
+public class RegisteredLearnersOrganisationDelete {
+    private final Long organisationalUnitId;
```

Comment:
Does this not need to be a list so that it deletes all of the child organisations too?